added short intros to section landing pages (#313)
elixir-europe/infectious-diseases-toolkit: data-description/index.md

title: Data description
---

> Data description involves detailing the datasets used in research or surveillance. This includes finding and adhering to relevant (meta)data standards and documentation practices to ensure data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and dependability of information regarding disease incidence, transmission, and treatment outcomes, which is necessary for effective public health interventions and scientific studies. It also promotes data sharing among various stakeholders, allowing for more metaanalyses and encouraging collaborations that can improve disease understanding and management.

Reviewer: ```suggestion
Data description involves detailing the datasets used in research or surveillance. This, for example, includes finding and adhering to relevant (meta)data standards and documentation practices to ensure data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and dependability of information regarding disease incidence, transmission, and treatment outcomes, which is necessary for effective public health interventions and scientific studies. Good data description promotes data sharing among various stakeholders, allowing for more meta-analyses and encouraging collaborations to improve disease understanding and management.
```

> Data description involves detailing the datasets used in research or surveillance. This, for example, includes finding and adhering to relevant (meta)data standards and documentation practices to ensure data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and dependability of information regarding disease incidence, transmission, and treatment outcomes, which is necessary for effective public health interventions and scientific studies. Good data description promotes data sharing among various stakeholders, allowing for more meta-analyses and encouraging collaborations to improve disease understanding and management.

Reviewer 2: ```suggestion
Data description involves detailing the datasets used in research or surveillance. This includes, for example, finding and adhering to relevant standards for (meta)data and documentation to ensure that  data are accurately represented, interoperable, and reusable. Accurate data description improves the clarity and reliability of information regarding disease incidence, transmission, and treatment outcomes, which is essential for ensuring effective public health interventions and scientific studies. Good data description promotes data sharing among various stakeholders, allowing for more meta-analyses, and encouraging collaborations to improve the understanding and management of disease.
```
